pkg/broker/variables.go

```diff
@@ -162,13 +162,13 @@ func ValidateVariables(parameters map[string]interface{}, variables []BrokerVari
 }
 
 func createJsonSchema(schemaVariables []BrokerVariable) map[string]interface{} {
-	var required []string
+	required := utils.NewStringSet()
 	properties := make(map[string]interface{})
 
 	for _, variable := range schemaVariables {
 		properties[variable.FieldName] = variable.ToSchema()
 		if variable.Required {
-			required = append(required, variable.FieldName)
+			required.Add(variable.FieldName)
```

Comment:
Is there any warnings we should do in case of an override? Or, should we document the behavior that later field definitions override earlier ones?